### PR TITLE
Feat disable user

### PR DESCRIPTION
**Pull request by iglimanaj**
This pull request introduces a few helper functions for dealing with users in Keycloak. The are handy if you need to enable or  disable users programmatically.

### CODE REVIEW

File: src/keycloak/keycloak_admin.py [hunk 1]
```diff
@@ -762,6 +762,46 @@ def update_user(self, user_id, payload):
...
+        """
+        return self.update_user(user_id=user_id, payload={"enabled": True})
+    
+    def disable_all_users(self):
+        """Disable all existing users.
+        """
+        users = self.get_users()
+        for user in users:
+            user_id = user["id"]
+            print(f"Disabling user with id: {user_id}")
```
Comment (ryshoooo) [inline src/keycloak/keycloak_admin.py]: No `print`s please :) I do not want to force unstoppable logs to the users of the library. You can use `logging` with a combination of a `verbose` parameter or just a `verbose` parameter itself keeping the prints, but the default should be 0 verbosity :)

File: src/keycloak/keycloak_admin.py [hunk 2]
```diff
@@ -762,6 +762,46 @@ def update_user(self, user_id, payload):
...
+            self.disable_user(user_id=user_id)
+        
+    def enable_all_users(self):
+        """Disable all existing users.
+        """
+        users = self.get_users()
+        for user in users:
+            user_id = user["id"]
+            print(f"Enabling user with id: {user_id}")
```
Comment (ryshoooo) [inline src/keycloak/keycloak_admin.py]: Same here

**Review by ryshoooo — COMMENTED** [review]
Hi @iglimanaj 

Thanks a lot for your contribution :) LGTM just a small comment on the `print` statements.

File: src/keycloak/keycloak_admin.py (hunk 1 above)
Comment (iglimanaj) [reply src/keycloak/keycloak_admin.py]: Good point. I removed the print statements.

**Comment by ryshoooo** [comment]
Looks like the pipeline is failing due to incorrectly formatted code. Make sure that `tox -e check` passes.

**Comment by iglimanaj** [comment]
@ryshoooo I just pushed the refactored code. Thanks for your review!

**Pull request merged**